test/integration/file-uploads/request-body.test.js

```diff
@@ -645,4 +645,81 @@ describe('file upload in request body', function () {
                 .to.equal('Binary file load error: file resolver interface mismatch');
         });
     });
+
+    (IS_BROWSER ? describe.skip : describe)('large file upload in request body', function () {
+        // eslint-disable-next-line mocha/no-sibling-hooks
+        before(function (done) {
+            this.run({
+                fileResolver: {
+                    stat: function (src, cb) {
+                        cb(null, {isFile: function () { return true; }, mode: 33188});
+                    },
+                    createReadStream: function () {
+                        // creating buffer of size 52428800 bytes corresponds to 50 MB
+                        return Buffer.alloc(52428800);
```

Comment:
Why this magic number? 
```suggestion
                        return Buffer.alloc(50 * 1024 * 1024);
```